manifests/v/Vendicated/Vencord/location/1.2.10/Vendicated.Vencord.location.installer.yaml

```
@@ -0,0 +1,20 @@
+
+PackageIdentifier: Vendicated.Vencord.location
+PackageVersion: 1.2.10
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Discord.Discord
+Installers:
+- Architecture: x64
```

```suggestion
- Architecture: x86
```
This package wouldn't run until I changed this in the VM's local manifest.